src/hooks/useHostedSession/useHostedSession.ts

```diff
@@ -111,17 +116,26 @@ export const useHostedSession = (): UseHostedSessionHook => {
   }, [client, onHostedSessionExpired.data])
 
   if (loading) {
-    return { loading: true }
+    return { loading: true, customTheme: CustomThemeApiField.parse({}) }
   }
 
   if (error) {
-    return { loading: false, error: error.message, refetch }
+    return {
+      loading: false,
+      error: error.message,
+      refetch,
+      customTheme: CustomThemeApiField.parse({}),
```

Comment:
maybe do 
`const defaultTheme = CustomThemeApiField.parse({}) `
and than use that in error, loading